### PR TITLE
Translation

### PR DESCRIPTION
**Pull request by steins-code**
New updates from translation

### CODE REVIEW

**Review by haosap — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -678,7 +676,7 @@
 
 "App_Information_Contact_Hotline_Description" = "Unser technischer Kundenservice ist für Sie da. Die Wartezeit ist weniger als 5 Minuten im Moment.";
 
-"App_Information_Contact_Hotline_Terms" = "Unsere Öffnungszeiten:\nMo - Fr: 08:00 - 22:00 Uhr\nSa - So: 10:00 - 22:00 Uhr\nEs gelten die Preise Ihres Telefonanbieters.";
+"App_Information_Contact_Hotline_Terms" = "Unsere Öffnungszeiten:\nMo - SA: 08:00 - 22:00 Uhr\n(außer an bundesweiten Feiertagen)\n";
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:679]: nope, should be `Mo - Sa:`

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -19,6 +19,9 @@
 
 "General_BackButtonTitle" = "Back";
 
+/* Accessibility */
+"AccessibilityLabel_Close" = "Schließen";
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:23]: Should be English.

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 3]
```diff
@@ -38,6 +41,8 @@
 
 "ExposureDetection_RefreshingIn" = "Update in %02d:%02d minutes";
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:42]: maybe add non-breakable space here, same goes for the German file

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 4]
```diff
@@ -507,6 +514,8 @@
 
 "Home_Risk_No_Date_Title" = "Unknown";
 
+"Home_Risk_Period_Update_Title" = "Updated every %@ hours";
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:517]: non-breakable-space? same for German file

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 5]
```diff
@@ -113,22 +110,22 @@
 
 "Settings_Reset_Label" = "Anwendung zurücksetzen";
 
-"Settings_Tracing_Description" = "Erlauben Sie die Aufzeichnung und Weitergabe von Covid-19-Zufallscodes.";
+"Settings_Tracing_Description" = "Erlauben Sie die Aufzeichnung und Weitergabe von COVID-19-Zufallscodes.";
 
-"Settings_Notification_Description" = "Erlauben Sie automatische Mitteilungen zu Covid-19-Risikostatus.";
+"Settings_Notification_Description" = "Erlauben Sie automatische Mitteilungen zu COVID-19-Risikostatus.";
```
Comment (steins-code) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:115]: needs to be changed

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings [hunk 6]
```diff
@@ -657,11 +672,11 @@
 
 "App_Information_Contact_Hotline_Text" = "+49 (0)800 7540001";
 
-"App_Information_Contact_Hotline_Number" = "+49 (0)800 7540001";
+"App_Information_Contact_Hotline_Number" = "+498007540001";
 
 "App_Information_Contact_Hotline_Description" = "Our technical customer service is there for you. Waiting time is currently less than 5 minutes.";
 
-"App_Information_Contact_Hotline_Terms" = "Business hours:\nMonday to Friday: 8am - 10pm\nSaturday and Sunday: 10am - 10pm\nThe standard charges of your telephone provider apply.";
+"App_Information_Contact_Hotline_Terms" = "Business hours:\nMonday to Saturday: 8am - 10pm\n(except national holidays)";
```
Comment (EisvogelSAP) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:679]: why is this not abbreviated like in the German version?

**Review by eisvogel — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings (hunk 2 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings]: done

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 1 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings]: done

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings (hunk 6 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:679]: Please ask translation folks

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings (hunk 4 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings:517]: done

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings (hunk 3 above)
Comment (steins-code) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings]: done

**Review by eisvogel — APPROVED** [review]

**Pull request merged**